packages/hub/src/lib/commit.ts

```diff
@@ -89,11 +99,40 @@ async function* commitIter(params: CommitParams): AsyncGenerator<unknown, Commit
 
 	const lfsShas = new Map<string, string | null>();
 
-	const gitAttributes = (
-		params.operations.find((op) => isFileOperation(op) && op.path === ".gitattributes") as CommitFile | undefined
-	)?.content;
+	const allOperations = await Promise.all(
+		params.operations.map(async (operation) => {
+			if (operation.operation === "addOrUpdate") {
```

Comment:
Actually here (and l.105 on the second if), you could use a guard to save 2 levels of indentation